pkg/auth/tokens/manager.go

```diff
@@ -630,7 +630,7 @@ var uaBackoff = wait.Backoff{
 
 func (m *Manager) NewLoginToken(userID string, userPrincipal v3.Principal, groupPrincipals []v3.Principal, providerToken string, ttl int64, description string) (v3.Token, string, error) {
 	provider := userPrincipal.Provider
-	if (provider == "github" || provider == "azuread" || provider == "googleoauth") && providerToken != "" {
+	if (provider == "github" || provider == "azuread" || provider == "googleoauth" || provider == "oidc" || provider == "keycloakoidc") && providerToken != "" {
```

Comment:
nit: can you make this commit more descriptive, or add a comment around the code. I've never understood it.

Reply:
I added a comment